test/e2e/specs/site-editor/title.test.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editorTitle: async ( { page, pageUtils, requestUtils }, use ) => {
+		await use( new EditorTitle( { page, pageUtils, requestUtils } ) );
+	},
+} );
+
+test.describe( 'Title', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+		await Promise.all( [
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
```

Comment:
I don't know that deleting templates is actually necessary. The puppeteer tests did it, but maybe the author was worried about other tests that modified templates.